validation-test/Reflection/reflect_nested.swift

```diff
@@ -43,6 +43,37 @@ reflect(object: obj)
 // CHECK-32-NEXT:   (bound_generic_class reflect_nested.OuterGeneric.Inner
 // CHECK-32-NEXT:     (bound_generic_class reflect_nested.OuterGeneric
 // CHECK-32-NEXT:       (struct Swift.Int))))
+  
+class OuterNonGeneric {
+  class InnerGeneric<T, U> {
+    var x: T
+    var y: U
+
+    init(x: T, y: U) {
+      self.x = x
+      self.y = y
+    }
+  }
+}
+
+var obj2 = OuterNonGeneric.InnerGeneric(x: 17, y: "hello")
+reflect(object: obj2)
+
+// CHECK-64: Reflecting an object.
+// CHECK-64: Instance pointer in child address space: 0x{{[0-9a-fA-F]+}}
+// CHECK-64: Type reference:
+// CHECK-64: (bound_generic_class reflect_nested.OuterNonGeneric.InnerGeneric
+// CHECK-64-NEXT:   (struct Swift.Int)
+// CHECK-64-NEXT:   (struct Swift.String)
+// CHECK-64-NEXT:     (bound_generic_class reflect_nested.OuterNonGeneric))
```

Comment:
This is wrong.   `OuterNonGeneric` is not a `bound_generic_class`, and making it so throws off RemoteMirror's layout calculations (which this test does not try to verify).